gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-server/src/main/java/org/apache/gobblin/service/FlowConfigsResource.java

```diff
@@ -140,7 +139,7 @@ public UpdateResponse delete(ComplexResourceKey<FlowId, EmptyRecord> key) {
    * @param originalFlowConfig original flow config to find original requester
    * @param requesterList list of requesters for this request
    */
-  public static void checkRequester(FlowConfig originalFlowConfig, List<ServiceRequester> requesterList) {
```

Comment:
Why not leave this as static to avoid code duplication? You could move it to FlowConfigsV2Resource if you're considering that FlowConfigsResource may be removed eventually

Reply:
Had to change it, coz using non static object (this.requesterService) inside.
Will work actively towards deprecating V1

Reply:
You could pass the requesterservice as an argument. But sure, I guess having the duplication is fine if we're probably not going to make changes to v1 after this.